### PR TITLE
feat(i18n-JA): adds Japanese language

### PR DESCRIPTION
**Pull request by Suisyan**
Add new Japanese translation.

Signed-off-by: Suisyan <99475190+Suisyan@users.noreply.github.com>

### CODE REVIEW

**Comment by pedrolamas** [comment]
Hi @Suisyan, thank you for your contribution!

Can you please add an entry for this here:

https://github.com/fluidd-core/fluidd/blob/999af77fb956dc8a8d3cfb71d3fe4b584b5bfdf7/src/store/config/index.ts#L18-L33

**Comment by pedrolamas** [comment]
There are also 3 empty translations, it would be great if you could add those also!

image

Thank you!!

**Comment by pedrolamas** [comment]
Please be aware that we will not be able to merge this until you sign your commit or change the PR main comment to include a `Signed-off-by: Your Name <your email address>` footer, as per [contribution guidelines](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md)!

**Review by pedrolamas — APPROVED** [review]
LGTM

File: src/locales/ja.yaml [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+      total: 合計
+      total_filament: フィラメント総使用量
+      total_filament_avg: プリントあたりの平均値
+      total_jobs: 印刷ジョブの総数
+      total_print_time: 総印刷時間
+      total_print_time_avg: プリントあたりの平均値
+      total_time: 合計時間
+      total_time_avg: プリントあたりの平均値
+      uncategorized: 未分類
+      unretract_extra_length: Unretract Extra Length
```
Comment (matmen) [inline src/locales/ja.yaml:229]: see above

File: src/locales/ja.yaml [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+      unretract_speed: Unretract Speed
```
Comment (matmen) [inline src/locales/ja.yaml:230]: see above

File: src/locales/ja.yaml [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+    camera_rotate_options:
+      '90': 90°
+      '180': 180°
+      '270': 270°
+      none: なし
+    camera_type_options:
+      iframe: HTTPページ
+      mjpegadaptive: MJPEG Adaptive
+      mjpegstream: MJPEG Stream
+      video: IP Camera
```
Comment (matmen) [inline src/locales/ja.yaml:348]: Not sure if a translation is plausible for these 3, as they are very specific

File: src/locales/ja.yaml [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+    btn:
+      check_for_updates: アップデートを確認
+      finish: 終了
+      update: アップデート
+      view_versions: バージョンを表示
+    label:
+      commit_history: 書込み履歴
+      commits_on: 書込みする
+      committed: 書込みする
+      dirty: DIRTY
```
Comment (matmen) [inline src/locales/ja.yaml:463]: See first comment

**Review by matmen — COMMENTED** [review]
Thanks for your contribution! There are a couple lines that are still untranslated, if possible please also translate those or let me know if translations for them are not plausible.

**Comment by Suisyan** [comment]
> Thanks for your contribution! There are a couple lines that are still untranslated, if possible please also translate those or let me know if translations for them are not plausible.


We will correct this information, including any mistranslations, after it is published.

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

**Pull request merged**